sdk/src/beta9/config.py

```diff
@@ -174,15 +175,25 @@ def prompt_for_config_context(
 
     try:
         while not (name := prompt_name()) or not isinstance(name, str):
-            pass
-
-        while not (gateway_host := prompt_gateway_host()) or not validate_ip_or_dns(gateway_host):
-            pass
-
-        while not (gateway_port := prompt_gateway_port()) or not validate_port(gateway_port):
-            pass
-
-        token = terminal.prompt(text="Token", default=None)
+            terminal.warn("Name is invalid.")
+
+        if settings.use_defaults_in_prompt:
+            gateway_host = DEFAULT_GATEWAY_HOST
+            gateway_port = DEFAULT_GATEWAY_PORT
+        else:
+            while not (gateway_host := prompt_gateway_host()) or not validate_ip_or_dns(
+                gateway_host
+            ):
+                terminal.warn("Gateway host is invalid.")
+
+            while not (gateway_port := prompt_gateway_port()) or not validate_port(gateway_port):
+                terminal.warn("Gateway port is invalid.")
+
+        if require_token:
+            while not (token := terminal.prompt(text="Token", default=None)) or len(token) < 64:
```

Comment:
64 is completely arbitrary. Is there a length that's guaranteed or should I keep it at less than what it could be (at least what I see in my account).

Reply:
I think this should be 128